x-pack/plugins/cases/public/components/all_cases/index.tsx

```diff
@@ -0,0 +1,607 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useCallback, useEffect, useMemo, useRef, useState } from 'react';
+import {
+  EuiBasicTable as _EuiBasicTable,
+  EuiContextMenuPanel,
+  EuiEmptyPrompt,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLoadingContent,
+  EuiProgress,
+  EuiTableSortingType,
+} from '@elastic/eui';
+import { EuiTableSelectionType } from '@elastic/eui/src/components/basic_table/table_types';
+import { isEmpty, memoize } from 'lodash/fp';
+import styled, { css } from 'styled-components';
+import classnames from 'classnames';
+
+import * as i18n from './translations';
+import { CaseStatuses, CaseType } from '../../../common/api';
```

Comment:
you should just be able to import from just `'../../../common';` since we have `export * from './api';`. I got yelled at in CI for anything that wasnt just `/common`, it must like you better.

Reply:
Yep, updated! It only yells when you're importing from an external plugin. So from security solution, but local imports within the plugin like this should be fine. Either way, I much prefer shorter import paths 😄